src/utils/index.js

```diff
@@ -26,4 +28,28 @@ module.exports = {
     if (!isString(str)) return str;
     return str.toLowerCase();
   },
+
+  /**
+   * Returns the lowercased string if the string is valid.
+   * @param {string} str String to lowercase
+   * @return {string} Lowercased string
+   */
+  getAndInsertNames: async (address, DBHelper) => {
+    if (isUndefined(address)) return;
+    let addresses = address;
+    if (!isArray(address)) addresses = [address];
+    const toCheckAddresses = [];
+    await eachOfSeries(addresses, async (item, index) => {
```

Comment:
this doesn't need to be in series since theres no limit to the number of queries for the db (at least that I know of). the ~40 req/second limit only applies to calls to the client server, ie. contract calls.